e2e/audited-regressions.spec.ts

```
@@ -1,63 +1,71 @@
-import { expect, test } from "@playwright/test";
+import { expect, test, type Page } from "@playwright/test";
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)